pkg/query-service/app/clickhouseReader/reader.go

```diff
@@ -3386,6 +3391,14 @@ func isDashboardWithTSV2(data map[string]interface{}) bool {
 	return strings.Contains(string(jsonData), "time_series_v2")
 }
 
+func isDashboardWithLogsClickhouseQuery(data map[string]interface{}) bool {
+	jsonData, err := json.Marshal(data)
+	if err != nil {
+		return false
+	}
+	return strings.Contains(string(jsonData), "signoz_logs.distributed_logs")
```

Comment:
I have seen some people just use `signoz_logs.logs`, so let's not assume everyone uses a distributed table.